src/components/NcAppSettingsDialog/NcAppSettingsDialog.vue

```diff
@@ -75,9 +75,39 @@ export default {
 ```
 </docs>
 
+<template>
+	<NcDialog v-if="open"
+		:additional-trap-elements="additionalTrapElements"
+		:container="container"
+		:name="name"
+		:navigation-classes="['app-settings__navigation']"
+		class="app-settings"
+		size="large"
```

Comment:
shouldnt that better be small by default?

Reply:
It is unchanged, it was previously also `large`. Otherwise the navigation would be never shown.

Reply:
what navigation? You mean the buttons at the right? Shouldnt they be positioned below the question?

Reply:
No this is the app settings dialog, its about the sections on the left.
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/1855448/82e3fb68-690b-406c-a610-4b93bbc8b746)